src/components/organisms/Footer/Links.module.css

```diff
@@ -0,0 +1,46 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  gap: calc(var(--spacer) * 2);
+  width: 100%;
+}
+
+.section {
+  text-align: left;
+}
+
+.title {
+  margin-bottom: calc(var(--spacer) / 2);
+  text-transform: uppercase;
```

Comment:
suggestion: maybe we can try `--font-weight-bold` and `--color-secondary` on the titles